### PR TITLE
convert documentation to Antora module

### PR DESCRIPTION
**Pull request by slonopotamus**
This is a first step to publish documentation on docs.asciidoctor.org

### CODE REVIEW

**Comment by slonopotamus** [comment]
Accompanying PR: https://github.com/asciidoctor/docs.asciidoctor.org/pull/60

**Comment by slonopotamus** [comment]
@mojavelinux Could you please take a look and confirm that I am using Antora stuff properly?

I'm not currently going to work on documentation restructuring. I'd prefer to make this a separate step after publishing process is properly set up.

File: docs/antora.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: epub3
```
Comment (Mogztter) [inline docs/antora.yml:1]: I think it should be `epub3-converter` if we want to stay consistent with `reveal.js-converter`

Comment (slonopotamus) [reply docs/antora.yml]: You're right, fixed.

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,743 @@
+= {project-name}: A _native_ EPUB3 converter for AsciiDoc
```
Comment (Mogztter) [inline docs/modules/ROOT/pages/index.adoc:1]: The title of the index page should be:

```suggestion
= Asciidoctor EPUB3 Documentation
```

If you follow this convention, the breadcrumb will display: `Asciidoctor EPUB3 > Page Title`.
You should also add the attribute `:navtitle: Introduction`.

Comment (slonopotamus) [reply docs/modules/ROOT/pages/index.adoc]: Thanks, updated PR accordingly.

**Pull request merged**